tools/config.py

```diff
@@ -193,7 +193,7 @@ def generate_config(path):
 
   config_data = utils.read_file(path_from_root('tools/config_template.py'))
   config_data = config_data.splitlines()[3:] # remove the initial comment
-  config_data = '\n'.join(config_data)
+  config_data = '\n'.join(config_data) + '\n'
```

Comment:
I thought you are opposed to trailing newlines in files? :smile:

Reply:
This is just the newline of the final line :)

There is only one thing worse than a file with trailing newlines and that is a text file that don't end in a newline at all!